node/narwhal/examples/README.md

```diff
@@ -19,35 +19,47 @@ cargo run --release -- example monitor
 
 ## Development
 
+```
+Usage: simple_node [OPTIONS] --mode <MODE> --node-id <ID> --num-nodes <N>
+
+Options:
+      --mode <MODE>                   The mode to run the node in [possible values: narwhal, bft]
```

Comment:
these docs could use a bit of information on what the difference is between these modes